README.md

```diff
@@ -112,8 +112,29 @@ spec:
   repositoryPath: sys-kube-applier
   runInterval: 3600
   serverSideApply: false
+  strongboxKeyringSecretRef: ""
 ```
 
+See the documentation on the Application CRD
+[spec](https://godoc.org/github.com/utilitywarehouse/kube-applier/apis/kubeapplier/v1alpha1#ApplicationSpec)
+for more details.
+
+#### Integration with `strongbox`
+
+[strongbox](https://github.com/uw-labs/strongbox) is an encryption tool, geared
+towards git repositories and working as a git filter.
+
+If the Application spec contains a `strongboxKeyringSecretRef` value, the value
+should be the name of a Secret resource (in the namespace where the Application
+resides), which contains a key named `.strongbox_keyring` with its value being
+a valid strongbox keyring file. That keyring is subsequently used when applying
+the Application, allowing for decryption of files under the `repositoryPath`.
+
+The secret containing the strongbox keyring should itself be version controlled
+to prevent kube-applier from pruning it. However, since it is a secret itself
+and would need be encrypted as well in git, it must be created manually the
+first time (or after any changes to its contents).
```

Comment:
it's (I think, @ribbybibby might need to correct me)

Reply:
Oops, didn't refresh before merging, sorry!
But I believe it is "its" (possessive adjective)